xrtpy/image_correction/tests/test_remove_lightleak.py

```diff
@@ -1,61 +1,34 @@
 from pathlib import Path
 
+import numpy as np
 import pytest
+from astropy.utils.data import get_pkg_data_path
 from sunpy.map import Map
 
 from xrtpy.image_correction.remove_lightleak import remove_lightleak
 
-
-def get_IDL_data_file():
-    """
-    The XRT composite fits file that been lightleak corrected in
-    IDL have been rename to begin with "ll" referring to lightleak.
-    """
-    directory = (
-        Path(__file__).parent.absolute()
-        / "data"
-        / "light_leak_testing_data_files"
-        / "IDL_lightleak_corrected_data_test_files"
+data_dir = Path(
+    get_pkg_data_path(
+        "data/light_leak_testing_data_files", package="xrtpy.image_correction.tests"
     )
-    data_files = directory.glob("ll_comp_XRT*.fits")
-    return sorted(data_files)
-
-
-IDL_filenames = get_IDL_data_file()
-
-
-def get_composite_data_files():
-    """
-    The XRT composite fits file are no corrected in IDL.
-    These files will be corrected using XRTpy.
-    """
-    directory = (
-        Path(__file__).parent.absolute()
-        / "data"
-        / "light_leak_testing_data_files"
-        / "xrtpy_lightleak_data_test_files"
-    )
-    data_file = directory.glob("comp_XRT*.fits")
-    return sorted(data_file)
-
-
-composite_filenames = get_composite_data_files()
-
-# Using zip as an iterator to pair the data files together. Trouble-free method to use in pytest-parametrize
-data_files = list(zip(IDL_filenames, composite_filenames, strict=False))
-
-
-@pytest.mark.parametrize(("idlfile", "compfile"), data_files)
-def test_lightleak(idlfile, compfile, allclose):
+)
+composite_filenames = sorted(
+    (data_dir / "xrtpy_lightleak_data_test_files").glob("comp_XRT*.fits")
+)
+IDL_filenames = sorted(
```

Comment:
This is a really personal and minor nick pick of mine. I would either lowercase IDL or make the entire variable upper case.

Reply:
I tend to agree, but I'll leave this to a separate PR if we want to make that change everywhere.